src/AppBundle/Controller/TiqrAppApiController.php

```diff
@@ -65,15 +80,24 @@ public function metadataAction(Request $request)
     {
         $key = $request->get('key');
         if (empty($key)) {
+            $this->logger->info('without enrollment');
```

Comment:
Maybe it's better to make this one and "with key" a little more verbose. A log with a message "with key" doesn't say much.

Reply:
👍